### DISCUSSION TITLE
Notification Center seems broken in general

### DISCUSSION THREAD

**Discussion by robertmaxton42** [valinet/ExplorerPatcher#2094] [General]
This is most obvious with Discord, which I use quite frequently:

* After installing Explorer Patcher and rebooting, Discord notifications make their usual noise, but...
* ... don't seem to show up in the notification center (which claims "no new notifications")
* don't bring up the taskbar
* don't show the push/toast popup with the message preview
* don't seem to translate to the usual orange 'alert' shading, either.

I looked through the settings twice, but I didn't see anything that look liked it could be causing the issue; I checked through Windows 11 settings as well, though it's harder to be sure there. 

Uninstalling ExplorerPatcher seems to have fixed the problem.

**Comment by pyrates999** [comment]
Please post windows version and build.  You can find this by running winver.
Please post explorer patcher version.

Please include what settings you changed in explorer patcher properties application.
